src/index.html

```diff
@@ -21,8 +21,8 @@
       <img id="logo" src="#" alt="logo" />
       <nav>
         <ul class="nav-list">
-          <li><a href="#"></a>Soccers</li>
-          <li><a href="#"></a>Leagues</li>
+          <li><a href="#" class="soccer-count">Soccers</a></li>
```

Comment:
Please assign numbers to 'Leagues' instead of  'Soccers' to make the page more relevant.